app/javascript/App.jsx

```diff
@@ -29,6 +29,29 @@ export default function App() {
   const currentUser = useAuth();
   const location = useLocation();
 
+  // check for the maintenance banner
+  const maintenanceBanner = useSiteSetting(['Maintenance']);
+  const notify = () => {
+    if (maintenanceBanner.data) {
+      toast.info(maintenanceBanner.data, {
+        position: 'top-center',
+        autoClose: false,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+        theme: 'light',
+        className: 'text-center',
+      });
+    }
+  };
```

Comment:
Do we need the separate function? Can this not just go inside the `useEffect`?

Reply:
No the separate function is not needed moved it inside useEffect!